wsd/DocumentBroker.cpp

```diff
@@ -4614,6 +4616,8 @@ bool DocumentBroker::lookupSendClipboardTag(const std::shared_ptr<StreamSocket>
         LOG_INF("Found and queued clipboard response for send of size " << FileUtil::Stat(clipFile->_file).size());
         return true;
     }
+#else
+    (void)type;
```

Comment:
In the interest of reducing `#ifdef` directives (which are quite noisy/distracting, we can use `[[maybe_unused]]` on the parameter `type`.